packages/account/tests/account.e2e.spec.ts

```diff
@@ -166,6 +178,50 @@ describe("Account Tests", () => {
     expect(newBalance - balance).toBe(1n);
   }, 60000);
 
+  it("Should throw and error if missing field for ERC20 Paymaster user op", async () => {
+    const nftAddress: Hex = "0x1758f42Af7026fBbB559Dc60EcE0De3ef81f665e";
+    const {
+      whale: { viemWallet: signer, publicAddress: recipient },
+      bundlerUrl,
+      biconomyPaymasterApiKey,
+    } = mumbai;
+
+    const smartWallet = await createSmartAccountClient({
+      signer,
+      bundlerUrl,
+      biconomyPaymasterApiKey,
+    });
+
+    const encodedCall = encodeFunctionData({
+      abi: parseAbi(["function safeMint(address _to)"]),
+      functionName: "safeMint",
+      args: [recipient],
+    });
+
+    const transaction = {
+      to: nftAddress, // NFT address
+      data: encodedCall,
+    };
+
+    const userOp = await smartWallet.buildUserOp([transaction]);
+
+    const feeQuotesResponse: FeeQuotesOrDataResponse = await (
+      smartWallet.paymaster as IHybridPaymaster<PaymasterUserOperationDto>
+    ).getPaymasterFeeQuotesOrData(userOp, {
```

Comment:
wish we could solve this also by adding another tokenTaymaster instance or something in SDK

Reply:
yes I mentioned [this](https://github.com/bcnmy/biconomy-client-sdk/issues/303) in dsm yesterday